src/v/config/tests/throughput_control_group_test.cc

```diff
@@ -195,17 +195,17 @@ SEASTAR_THREAD_TEST_CASE(throughput_control_group_by_principal_test) {
 #    - name: (future) match a group (principal 'group')
 #      principals:
 #        - group: greek_letters
-#    - name: (future) match schema registry (principal 'ephemeral user')
-#      principals:
-#        - service: schema registry
-#    - name: (future) match PP (principal 'ephemeral user')
-#      principals:
-#        - service: panda proxy
-#    - name: (future) match heterogeneous set of principals
-#      principals:
-#        - user: servicebot
-#        - service: schema registry
-#        - service: panda proxy
+    - name: match schema registry (principal 'ephemeral user')
+      principals:
+        - service: schema registry
```

Comment:
Question: Why not keep this as `security::principal_type`? I.e., `ephemeral_user` rather than `service`?

I don't dislike it, but it adds an overloaded term.

Reply:
That would have exposed an implementation detail for the users. Currently the name of the ephemeral user used for SR/PP is not exposed anywhere, and is only a matter of the code. It can be e.g. a uuid as well, and now it can be changed anytime.